src/librustc_trans/mir/analyze.rs

```diff
@@ -160,8 +160,9 @@ impl<'mir, 'bcx, 'tcx> Visitor<'tcx> for LocalAnalyzer<'mir, 'bcx, 'tcx> {
                 LvalueContext::Call => {
                     self.mark_assigned(index);
                 }
-                LvalueContext::Consume => {
-                }
+                LvalueContext::StorageLive |
+                LvalueContext::StorageDead |
+                LvalueContext::Consume => {}
```

Comment:
Nit: In this context, I strongly prefer the `}` to go on the next line, to break this group of arms up from other groups when scanning visually.